xbmc/windowing/windows/WinEventsWin32.cpp

```diff
@@ -9,39 +9,41 @@
 #ifndef _USE_MATH_DEFINES
 #define _USE_MATH_DEFINES
 #endif
-#include <dbt.h>
-#include <math.h>
-#include <Shlobj.h>
-#include <Windowsx.h>
+#include "WinEventsWin32.h"
 
-#include "Application.h"
-#include "AppInboundProtocol.h"
+#include "ServiceBroker.h"
+#include "Util.h"
+#include "WinKeyMap.h"
+#include "application/AppInboundProtocol.h"
+#include "application/Application.h"
 #include "guilib/GUIComponent.h"
-#include "guilib/GUIControl.h"       // for EVENT_RESULT
+#include "guilib/GUIControl.h" // for EVENT_RESULT
 #include "guilib/GUIWindowManager.h"
+#include "input/InputManager.h"
 #include "input/mouse/MouseStat.h"
 #include "input/touch/generic/GenericTouchActionHandler.h"
 #include "input/touch/generic/GenericTouchSwipeDetector.h"
-#include "input/InputManager.h"
 #include "messaging/ApplicationMessenger.h"
 #include "network/Zeroconf.h"
 #include "network/ZeroconfBrowser.h"
-#include "platform/win32/CharsetConverter.h"
-#include "platform/win32/powermanagement/Win32PowerSyscall.h"
-#include "platform/win32/storage/Win32StorageProvider.h"
-#include "platform/win32/WIN32Util.h"
 #include "peripherals/Peripherals.h"
-#include "ServiceBroker.h"
+#include "rendering/dx/RenderContext.h"
 #include "storage/MediaManager.h"
-#include "Util.h"
 #include "utils/JobManager.h"
-#include "utils/log.h"
 #include "utils/StringUtils.h"
-#include "rendering/dx/RenderContext.h"
-#include "WinKeyMap.h"
-#include "WinEventsWin32.h"
+#include "utils/log.h"
+
+#include "platform/win32/CharsetConverter.h"
+#include "platform/win32/WIN32Util.h"
+#include "platform/win32/powermanagement/Win32PowerSyscall.h"
+#include "platform/win32/storage/Win32StorageProvider.h"
 
 #include <array>
+#include <math.h>
+
+#include <Shlobj.h>
+#include <Windowsx.h>
```

Comment:
Moving `<Windowsx.h>` seems has caused side effects because `IsMinimized` is defined inside and conflicts with:

https://github.com/xbmc/xbmc/blob/a65b73c530f01a6a285411f8ebe7929218dec015/xbmc/windowing/windows/WinEventsWin32.cpp#L615

Maybe a good solution is undefine with:

```c++
#ifdef IsMinimized
#undef IsMinimized
#endif
```

after include `<Windowsx.h>`   (insert this in line 50 or so)

Reply:
no wonder i couldn't find any references, it's windows itself conspiring with the clang-format monster of nonsense. thanks.